src/webapi/rendering_context.rs

```diff
@@ -27,7 +183,771 @@ impl RenderingContext for CanvasRenderingContext2d {
+    pub fn get_global_alpha(&self) -> f64 {
+        js! (
+            return @{&self.0}.globalAlpha
+        ).try_into().unwrap()
+    }
+
+    /// The CanvasRenderingContext2D.globalAlpha property of the Canvas 2D API specifies the alpha 
+    /// value that is applied to shapes and images before they are drawn onto the canvas. 
+    /// The value is in the range from 0.0 (fully transparent) to 1.0 (fully opaque).
+    /// 
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalAlpha)
+    pub fn set_global_alpha(&self, global_alpha: f64) {
+        assert!(global_alpha > 0 as f64 && global_alpha < 1 as f64);
+        js! { @(no_return)
+            @{&self.0}.globalAlpha = @{global_alpha};
+        }
+    }
+
+    /// The CanvasRenderingContext2D.globalCompositeOperation property of the Canvas 2D API sets the 
+    /// type of compositing operation to apply when drawing new shapes, where type is a string identifying 
+    /// which of the compositing or blending mode operations to use.
+    /// 
+    /// [(JavaScript docs)](https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/globalCompositeOperation)
+    pub fn get_global_composite_operation(&self) -> CompositeOperation {
+        let composite_operation_str: String = js! (
+            return @{&self.0}.globalCompositeOperation
+        ).try_into().unwrap();
+        match composite_operation_str.as_ref() {
+            "source-over" => CompositeOperation::SourceOver,
+            "source-in" => CompositeOperation::SourceIn,
+            "source-out" => CompositeOperation::SourceOut,
+            "source-atop" => CompositeOperation::SourceAtop,
+            "destination-over" => CompositeOperation::DestinationOver,
+            "destination-in" => CompositeOperation::DestinationIn,
+            "destination-atop" => CompositeOperation::DestinationAtop,
+            "lighter" => CompositeOperation::Lighter,
+            "copy" => CompositeOperation::Copy,
+            "xor" => CompositeOperation::Xor,
+            "multiply" => CompositeOperation::Multiply,
+            "screen" => CompositeOperation::Screen,
+            "overlay" => CompositeOperation::Overlay,
+            "darken" => CompositeOperation::Darken,
+            "lighten" => CompositeOperation::Lighten,
+            "color-dodge" => CompositeOperation::ColorDodge,
+            "color-burn" => CompositeOperation::ColorBurn,
+            "hard-light" => CompositeOperation::HardLight,
+            "soft-light" => CompositeOperation::SoftLight,
+            "difference" => CompositeOperation::Difference,
+            "exclusion" => CompositeOperation::Exclusion,
+            "hue" => CompositeOperation::Hue,
+            "saturation" => CompositeOperation::Saturation,
+            "color" => CompositeOperation::Color,
+            "luminosity" => CompositeOperation::Luminosity,
+            _ => panic!("Unexpected globalCompositeOperation value"),
```

Comment:
It'd be nice to print the value in the `panic!` message so *if* this happens we'll know what to add.